libs/SmartStore/SmartStore/Classes/SFSmartStore.m

```diff
@@ -889,13 +896,12 @@ - (NSString*) readFromEncryptedFile:(NSString*)filePath
 
 - (void) writeToEncryptedFile:(NSString*)filePath
                        content:(NSString *)content
-                          key:(NSData*)key
-                           iv:(NSData*)iv
+                       encKey:(SFEncryptionKey*)encKey
 {
     NSOutputStream *outputStream = nil;
-    if (key) {
+    if (encKey) {
         SFEncryptStream *encryptStream = [[SFEncryptStream alloc] initToFileAtPath:filePath append:NO];
```

Comment:
I could go further and change SFDecryptStream to work with SFEncryptionKey instead of its components.